### DISCUSSION TITLE
Box shadow designed for dark style

### DISCUSSION THREAD

**Discussion by aroberge** [pradyunsg/furo#222] [Ideas] [answered]
I just switched to using furo today ... and I really, really like its look.  Thank you very much for creating it and making it available.

Currently, admonitions have a box shadow that only shows in light style; it is also a very subtle visual effect, that helps to distinguish the content of an admonition with the surrounding environment since they use the same background color.

![image](https://user-images.githubusercontent.com/629698/131015611-73a88801-64cb-490f-8bfe-0f5daea05b5d.png)

With the dark theme, the box shadow is not visible, but the background colour of the admonition is slightly different from that of the surrounding text.

![image](https://user-images.githubusercontent.com/629698/131015631-f04395f6-fd5f-4ca4-a795-8a581b72bff0.png)

Personally, I like/prefer the visual effect given by using box shadow to distinguish elements.  Currently, the colors used for box shadow is a percentage of black  (`rgb(0 0 0 / d%`).  I'm wondering if it might be preferable to use a neutral grey instead, something like (`rgb(128 128 128 / d%`), so that it would work in both themes.

Here's what it looks on my site (where I experimented and increased the percentages, as my old eyes have trouble seeing subtle effects)

![image](https://user-images.githubusercontent.com/629698/131016831-5fbb0e8e-0fe8-4378-b40d-d3345f1c8279.png)

![image](https://user-images.githubusercontent.com/629698/131016849-3ca3055d-ad35-4348-a708-398492faf345.png)

Someone with better designing skills than me could certainly come up with something that would work well with both light and dark themes.

**Comment by pradyunsg** [comment, marked as answer]
I remember spending quite a lot of time on this, before settling on changing the background color.

I found it tricky to not have the admonitions to completely "steal" attention with the light-shadow-on-dark approach. A light shadow on dark background always looks out of place, and didn't feel right for auxiliary content like admonitions.

**Reply by aroberge** [reply]
Thank you; this makes a lot of sense.
